gpbft/gpbft.go

```diff
@@ -378,7 +386,11 @@ func (i *instance) receiveOne(msg *GMessage) (bool, error) {
 		return true, i.tryCommit(msg.Vote.Round)
 	}
 	// Try to complete the current phase in the current round.
-	return true, i.tryCurrentPhase()
+	if err := i.tryCurrentPhase(); err != nil {
+		return true, err
+	}
+
+	return true, i.tryRebroadcast()
```

Comment:
As far as I can tell from the FIP, this should only ever be triggered from the alarm timeout. I don't think we need it here.